### DISCUSSION TITLE
Can't push commits to the repository

### DISCUSSION THREAD

**Discussion by yyumeee** [community/community#129345] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm new to GitHub, and I'm trying to push a commit I made on GitHub Desktop onto the GitHub repository, but I always get this error message, despite seemingly being no connection issues. Has anyone experienced something similar? I already tried switching Wi-Fis but the error persists
![Screenshot (251)](https://github.com/community/community/assets/168559716/1b171a2e-bc83-429f-b5ac-860290546ac8)

**Comment by davevad93** [comment]
Hi @yyumeee ,

Take a look at this: [GitHub Desktop : The remote disconnected. Check your Internet connection and try again.](https://github.com/desktop/desktop/issues/14936)

If you're trying to push a large file (more than 2GB), It's very likely that GitHub is timing out your push due to the file size.

- [Troubleshooting the 2 GB push limit](https://docs.github.com/en/get-started/using-git/troubleshooting-the-2-gb-push-limit)

If that's your issue, you should consider using [Git LFS](https://git-lfs.com/). Here's a walkthrough from the docs:

1. [About large files on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)
2. [About Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage)
3. [Installing Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage)
4. [Configuring Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage)

**Comment by manuel14mds** [comment, marked as answer]
Look at this video, it's only five minutes.
https://www.youtube.com/watch?v=REBD-M1O_G4

**Comment by zecoryx** [comment]
Response #2

**Comment by yyumeee** [comment]
Thank you all for the help, it was indeed a matter of the push being too large.
